content/news/025/index.md

```diff
@@ -710,28 +689,12 @@ combines a vim-like modal interface with a cursor-based editor.
 - Community: [Discord](https://discord.gg/xHggPjfsS9)
 - Download: [rx-0.5.2-x86_64.AppImage][rx_download]
 
-Release v0.5 was just published with the following new features and updates:
-
-```markdown
-- New tool `:flood`, for filling an area with color
-- New brush mode `line`, for drawing straight lines
-- New command `:selection/flip <axis>` to flip the current selection
-- New command `:export <resize> <path>`, eg. `:export @2x meme.gif`
-- New command `:p/write <file>` to save the color palette to a file
-- New command `:p/sample` to sample view colors into palette
-- New command `:p/sort` to sort the palette colors
-- New command `:p/gradient` to add a color gradient to the palette
-- New setting `:p/height` to set the palette height in cells
-- Move between frames with `h` and `l` keys
-- Use *pan* tool in help mode
-- Fix command key ':' for non-ANSI layouts
-- Implement new command parser based on `memoir`
-- Add compatibility for non x86-64 systems
-- Support pasting from clipboard in command-line
-- Support character inputs in key mappings, eg. `map '?' :help`
-- Skip unreadable files when reading a directory
-- New, smoother animation rendering
-```
+Release v0.5 was just published. There are several new commands that help with
```

Comment:
It feels like this shortened version could be a simple list, like

- There are several new commands that help with color palettes.
- A flood fill tool has been added.
- You can now move between frames with the `h` and `l` keys.
- The command key `:` has been fixed to help with non-ANSI layouts.
- Compatability was added for non x86_64 systems.
- Support was added for pasting from the clipboard into the command line.
- Animation rendering is now a lot smoother.

Reply:
I think I'm going to leave it as a paragraph :ok_hand: